MobilityData/mobility-feed-api
# feat: migrate DEV PostgreSQL database to QA PostgreSQL instance

**Summary:**
To reduce cost, the DEV database has been moved from the DEV instance to the QA instance. The DEV instance is deleted and we only have the [QA instance containing two databases](https://console.cloud.google.com/sql/instances/mobilitydata-database-instance/overview?project=mobility-feeds-qa). 

Closes #210 
**Expected behavior:** 
The DEV functions and API now use the QA instance-based DEV database.

Explain and/or show screenshots for how you expect the pull request to work in your testing (in case other devices exhibit different behavior).

**Testing tips:**
- Visit our [Swagger UI](https://mobilitydata.github.io/mobility-feed-api/SwaggerUI/index.html) page
- Select the DEV server
- Authenticate using your access token
- Try an endpoint that queries the database like [`/v1/gtfs_feeds/{id}`](https://mobilitydata.github.io/mobility-feed-api/SwaggerUI/index.html#/feeds/getGtfsFeed)
- You should receive a 200 status code response

Please make sure these boxes are checked before submitting your pull request - thanks!
